mocha.mjs

```diff
@@ -0,0 +1,68 @@
+// this built file is not committed to the repo
+/* eslint-disable-next-line n/no-missing-import */
+import "./mocha.js";
+
+const {mocha, Mocha} = globalThis;
+
+const mochaExports = {
+  after: (...args) => globalThis.after?.(...args),
```

Comment:
I don't think Mocha should fail silently when the global `after` doesn't exist. That would indicate the page hasn't run `mocha.setup("bdd")` or an equivalent. Which would be something they'd want an error for.

```suggestion
  after: (...args) => after(...args),
```

In fact, I think it'd be useful to even throw a specific error:

```suggestion
  after: (...args) => {
    if (typeof after === "undefined") {
      throw new Error("`after` is not yet available. `mocha.setup("bdd\")` needs to be run first.");
    }
  },
```

WDYT?

Reply:
Good point! Yes, I think you're right. It's fine as-is. We can treat all of those suggestions as followup issues. 👍